pybikes/cyclopolis.py

```diff
@@ -98,3 +99,46 @@ def __init__(self, name, latitude, longitude, bikes, free, extra):
         self.bikes     = bikes
         self.free      = free
         self.extra     = extra
+
+
+class CyclopolisApi(BaseSystem):
+    sync = True
+
+    def __init__(self, tag, meta, feed_url):
+        super(CyclopolisApi, self).__init__(tag, meta)
+        self.feed_url = feed_url
+
+    def update(self, scraper=None):
+        scraper = scraper or PyBikesScraper()
+
+        data = json.loads(scraper.request(self.feed_url))
+
+        stations = []
+
+        self.stations = []
+        for station in data:
+            name = station["name"]
+            latitude = float(station["location"]["coordinates"][1])
+            longitude = float(station["location"]["coordinates"][0])
+            bikes = station["vehicle_docks_available"][0]["count"]
```

Comment:
this is ambiguous and might be only slots. i haven't seen more in the feed.

Reply:
Yes.. this seems like a direct port of a gbfs v3 feed, and `vehicle_docks_available` counts the number of docks per vehicle type. The number of bikes available should be under `num_vehicles_available` or `vehicle_types_available` which is empty, meaning one of the following:
* it is really empty
* they do not provide this information

I think we should set this as the number of `free` on the station, and maybe wait and see if `vehicle_types_available` comes alive at some point?

Reply:
Added handling for vehicle types and vehicle docks. 

```diff
-            bikes = station["vehicle_docks_available"][0]["count"]
+            bikes = sum(map(lambda vd: vd['count'], station["vehicle_types_available"]))
+            free = sum(map(lambda vd: vd['count'], station["vehicle_docks_available"]))
```

Will add an issue to check back in some weeks https://github.com/eskerda/pybikes/issues/801